authenticator/authenticator.go

```diff
@@ -212,9 +215,12 @@ func (a *Authenticator) getAuthRequest() (rest.Request, error) {
 		Nonce:          nonce,
 		Label:          fmt.Sprintf("%s-%d", labelPrefix, labelPostFix),
 		ReadOnly:       a.ReadOnly,
-		ExpirationTime: tokenExpiration,
+		ExpirationTime: a.TokenExpiration,
```

Comment:
This would be a get method that returns the token expiration as a string formatted in a way the api understands it.

Reply:
This could be pretty simple, with for example a fmt.Sprintf as 86400 seconds is also allowed :)

Reply:
Done